spec/acceptance/api/cms_section_spec.rb

```diff
@@ -8,6 +8,7 @@
 
     get '/admin/api/cms/sections.:format', action: :index do
       let(:collection) { provider.sections.order(:id) }
+      let(:serialized) { representer.send(serialization_format, short: true) }
```

Comment:
If `serialization_format` is not private, we should use `public_send`.

Reply:
Done